### ISSUE TITLE
[bugreport]: Commands not executed when executor type PLAYER_AND_CONSOLE is set

### ISSUE THREAD

**Issue by DSeeLPYT** [PXAV/kelp#27]
### What do we have to do to reproduce this bug? 
1. Create a new command with the executor PLAYER_AND_CONSOLE
2. allow custom parameters
3. put this in the onCommand(KelpPlayer,String[]) method:
```java
System.out.println(Arrays.toString(args));
```

### What would you expect to happen?
When the executorType is PLAYER_AND_CONSOLE and you execute the command with parameters, you see nothing in the console. But when the executorType is PLAYER_ONLY then everything works and you see the args in the console.

### Environment information
* Operating system: Windows 10 20H1
* Java-Version: 1.8
* Kelp-Core-Version: v.0.0.3
* Version-Implementation: 1.8

**Comment by PXAV** [comment]
Thank you for the bug report, I'll see how I can fix it!

**Comment by DSeeLPYT** [comment]
My Subcommands are not working too. The onCommand Method is not executed. But the onCommandRegister Method is executed but when i use get `getSubCommands().size()` in my parent command it returns 0.

**Comment by PXAV** [comment]
I have now fixed the bug. Actually, it was not a real bug, as `PLAYER_AND_CONSOLE` only called the `onCommand(ConsoleSender, args)` method and not the player method, because the command should be the same for players and consoles. But I have now changed it so that you can have both `onCommand` methods and depending on the executor, the corresponding method will be called.

If you want to have the same command handler method for both player and console, then you could do something like that:
```java
@CreateCommand(name = "helloworld", executorType = ExecutorType.PLAYER_AND_CONSOLE)
public class test extends KelpCommand {

  private KelpConsoleSenderFactory senderFactory;

  @Inject
  public test(KelpConsoleSenderFactory senderFactory) {
    this.senderFactory = senderFactory;
  }

  @Override
  public void onCommandRegister() {
    allowCustomParameters(true);
  }

  @Override
  public void onCommand(KelpConsoleSender consoleSender, String[] args) {
    System.out.println("command executed by either player or console " + Arrays.toString(args));
  }

  @Override
  public void onCommand(KelpPlayer player, String[] args) {
    CommandSender sender = player.getBukkitPlayer();
    this.onCommand(senderFactory.newKelpConsoleSender(sender), args);
  }

}
```

The fix will be released with v0.0.4
